manifest.json

```diff
@@ -27,51 +27,64 @@
 				"name": "domain",
 				"type": "domain",
 				"ask": {
-					"en": "Choose the domain name where Synapse is installed.",
-					"fr": "Choisissez le nom de domaine associé à Synapse."
+					"en": "Choose the domain name where the local Matrix/Synapse instance is installed.",
+					"fr": "Choisissez le nom de domaine associé à l'instance locale Matrix/Synapse."
 				},
 				"example": "domain.org"
 			},
 			{
 				"name": "whatsappbot",
 				"type": "string",
 				"ask": {
-					"en": "Choose the user name of the whatsapp bot for which to create a YunoHost and a local synapse server account.",
-					"fr": "Choisissez le nom d'utilisateur du robot whatsapp pour lequel un compte sera créé sur YunoHost et le serveur local synapse."
+					"en": "Choose the local Matrix/Synapse user name of the WhatsApp bot.",
+					"fr": "Choisissez le nom d'utilisateur local Matrix/Synapse du robot WhatsApp."
 				},
 				"example": "whatsappbot",
+				"help": {
+					"en": "Invite @whatsappbot:domain.org from an authorize Matrix account to setup the bridge.",
+					"fr": "Inviter @whatsappbot:domain.org depuis votre compte Matrix principal pour démarrer la passerelle."
+				}
```

Comment:
You are forgetting to add commas when you add new fields.

Reply:
oops, too much copy-pasting... Should I close the pull request, add a commit on my fork, and then resubmit a PR?
Or how should I proceed?

Reply:
I discovered that pushing to the master branch of my fork adds the commit to the PR. C'est quand meme bien fait!